Gemfile

```diff
@@ -9,7 +9,13 @@ gem 'nokogiri', '~> 1.6.0'
 group :development do
   gem 'jeweler', '~> 2.0.1'
 
-  # gem 'ruby-oci8', '~> 2.1.0'
-  gem 'ruby-oci8', :git => 'git://github.com/kubo/ruby-oci8.git', :platforms => :mri
+  platforms :ruby, :mswin, :mingw do
+    gem 'ruby-oci8', '~> 2.1.2'
```

Comment:
Maybe we should rather try with ruby-oci8 >= 2.1.2 (the current version seems to be 2.2.1).